cluster/addons/fluentd-elasticsearch/fluentd-es-image/Dockerfile

```diff
@@ -18,26 +19,21 @@ RUN ulimit -n 65536
 ENV DEBIAN_FRONTEND noninteractive
 
 # Install prerequisites.
-RUN apt-get update && \
-    apt-get install -y curl && \
-    apt-get install -y -q libcurl4-openssl-dev make && \
-    apt-get clean
+RUN apt-get update && apt-get install -y -q curl make g++ && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Install Fluentd.
-RUN /usr/bin/curl -L http://toolbelt.treasuredata.com/sh/install-ubuntu-trusty-td-agent2.sh | sh
+RUN curl -L https://td-toolbelt.herokuapp.com/sh/install-ubuntu-trusty-td-agent2.sh | sh
```

Comment:
Yeah this hasn't been kept up to date but will update soon & let you know when ready again. Thanks @a-robinson!